tests/test_csc.py

```diff
@@ -54,232 +55,253 @@ def test_standard_state_transitions(self):
+                        await cmd_attr.start(cmd_attr.DataType(), timeout=BASE_TIMEOUT)
+
+            # send start; new state is DISABLED
+            cmd_attr = getattr(harness.remote, f"cmd_start")
+            await asyncio.sleep(BASE_TIMEOUT)  # give time for event loop to run
+            harness.remote.evt_summaryState.flush()
+            await cmd_attr.start(timeout=120)  # this one can take longer to execute
+            state = await harness.remote.evt_summaryState.next(flush=False,
+                                                               timeout=BASE_TIMEOUT)
+            self.assertEqual(harness.csc.summary_state, salobj.State.DISABLED)
+            self.assertEqual(state.summaryState, salobj.State.DISABLED)
+
+            # Check that settings applied was published
+            settings = await harness.remote.evt_settingsApplied.next(flush=False,
+                                                                     timeout=BASE_TIMEOUT)
+            self.assertIsNotNone(settings)
+
+            for bad_command in commands:
+                if bad_command in ("enable", "standby"):
+                    continue  # valid command in DISABLED state
+                with self.subTest(bad_command=bad_command):
+                    cmd_attr = getattr(harness.remote, f"cmd_{bad_command}")
+                    with self.assertRaises(salobj.AckError):
+                        await cmd_attr.start(cmd_attr.DataType(), timeout=BASE_TIMEOUT)
+
+            # send enable; new state is ENABLED
+            cmd_attr = getattr(harness.remote, f"cmd_enable")
+            await asyncio.sleep(BASE_TIMEOUT)  # give time for the event loop to run
+            harness.remote.evt_summaryState.flush()
+            try:
+                # enable may take some time to complete
+                await cmd_attr.start(cmd_attr.DataType(), timeout=120.)
+            finally:
                 state = await harness.remote.evt_summaryState.next(flush=False,
                                                                    timeout=BASE_TIMEOUT)
-                self.assertEqual(harness.csc.summary_state, salobj.State.DISABLED)
-                self.assertEqual(state.summaryState, salobj.State.DISABLED)
-
-                # Check that settings applied was published
-                settings = await harness.remote.evt_settingsApplied.next(flush=False,
+            self.assertEqual(harness.csc.summary_state, salobj.State.ENABLED)
+            self.assertEqual(state.summaryState, salobj.State.ENABLED)
+
+            # check that position was published
+            ls = await harness.remote.evt_reportedLinearStagePosition.aget(timeout=BASE_TIMEOUT)
+            fw = await harness.remote.evt_reportedFilterPosition.aget(timeout=BASE_TIMEOUT)
+            gw = await harness.remote.evt_reportedDisperserPosition.aget(timeout=BASE_TIMEOUT)
+
+            self.assertIsNotNone(ls)
+            self.assertIsNotNone(fw)
+            self.assertIsNotNone(gw)
```

Comment:
I don't think `aget` can return None (unlike get). It can only raise if it takes too long to get data.